package.json

```diff
@@ -10,11 +10,14 @@
     "url": "https://github.com/PaulRBerg/prb-math/issues"
   },
   "devDependencies": {
+    "@forge/std": "github:foundry-rs/forge-std#e8a047e3f40f13fa37af6fe14e6e06283d9a060e",
```

Comment:
There's no need to use a faux `@forge` organization. We can just type it as `forge-std`.